trin-core/src/types/accumulator.rs

```diff
@@ -0,0 +1,203 @@
+use ethereum_types::U256;
+use ssz::Encode;
+use ssz_derive::{Decode, Encode};
+use ssz_types::{typenum, VariableList};
+use tree_hash::{MerkleHasher, TreeHash};
+use tree_hash_derive::TreeHash;
+
+use crate::types::header::Header;
+
+const EPOCH_SIZE: usize = 8192; // number of blocks / epoch
+const MAX_HISTORICAL_EPOCHS: usize = 131072; // 2 ** 17
+
+// ssz list[Hash256, max_length = MAX_HISTORICAL_EPOCHS]
```

Comment:
Let's make those doc comments with `///` and probably add some short descriptions.